app/views/research_sessions/questions/expenses.html.erb

```diff
@@ -1,43 +1,89 @@
-<div class="main-content">
-  <%= render 'shared/error_summary' %>
-  <h1 class="page-heading">Expenses</h1>
-  <%= barnardos_form_with model: @research_session, url: wizard_path do |form| %>
+<%= c('Step') do %>
+  <%= c('Title', text: 'Expenses') %>
 
-    <%= form.radio_group_vertical :expenses_enabled,
-      { true => 'Yes', false => 'No' },
-      { autofocus_first_item: true },
-      legend: t('helpers.legend.research_session.expenses_enabled')
-    %>
+  <%= c('Form', action: wizard_path, patch: true) do %>
+    <%= c('OfManyChoiceControl',
+      id: 'research_session[expenses_enabled]',
+      name:'expenses_enabled',
+      collection: "research_session[#{:expenses_enabled}]",
+      choices: [
+        {
+          id: 'expenses_enabled_true',
+          value: true,
+          label: 'Yes'
+        },
+        {
+          id: 'expenses_enabled_false',
+          value: false,
+          label: 'No' 
+        }
+      ],
+      choices_selected: @research_session.expenses_enabled.nil? ? [false] : [@research_session.expenses_enabled],
+      legend: t('helpers.legend.research_session.expenses_enabled'),
+      choice_type: 'single',
+      errored: @research_session.errors['expenses_enabled'].present?,
+      validation: @research_session.errors['expenses_enabled'][0]
+    ) %>
 
-    <div class="js-ConditionalSubfield" data-controlled-by="research_session[expenses_enabled]" data-control-value="true">
-      <div class="reactive-container">
-        <div class="reactive-container__form">
+  <%= c('ChoiceControlledContent',
+      is_active: (@research_session.expenses_enabled.present? && [@research_session.expenses_enabled.to_s].include?('true')),
+      controlled_by: '[id="research_session[expenses_enabled_true]"]') do %>
+        <%= c('SinglelineTextControl',
+          id: 'research_session[travel_expenses_limit]',
+          name: 'travel_expenses_limit',
+          value: @research_session.travel_expenses_limit,
+          label: t('research_session_attr_labels.travel_expenses_limit'),
+          hint: 'e.g. 0.00',
```

Comment:
Could we use a real world example here? i.e. `5.00`

Reply:
i've added this in but do we need anyone else to review what we can use as e.g as the old consent form just uses 0.00 as a placeholder